lib/include/hse/ikvdb/throttle.h

```diff
@@ -30,31 +84,39 @@ enum {
  */
 #define THROTTLE_DELAY_MAX           268435456  /*        500,000  bytes/sec */
 #define THROTTLE_DELAY_START_HEAVY     2097151  /*     64,000,030  bytes/sec */
-#define THROTTLE_DELAY_START_MEDIUM     731241  /*    183,547,869  bytes/sec */
+#define THROTTLE_DELAY_START_MEDIUM    1342177  /*    100,000,020  bytes/sec */
```

Comment:
Does the storage profile cli need to change at all?

Reply:
The storage profile tells you what throttle setting to use. I don't know. Just asking a question.

Reply:
Yeah, that's all mostly unnecessary with this patch, except for a time-zero problem where we have no idea how fast we can spill to cN, yet we can ingest into c0 at memory speeds.  That that stuff now basically exists to prevent those ginormous spikes in c0 ingest...